src/controllers/moderationactions/ModerationAction.cpp

```diff
@@ -124,6 +100,12 @@ ModerationAction::ModerationAction(const QString &action)
         this->line1_ = xD.mid(0, 2);
         this->line2_ = xD.mid(2, 2);
     }
+
+    if (!iconPath.isEmpty())
+    {
+        this->builtInImageToLoad_ = BuiltInImage::None;
```

Comment:
I think since user-specified images are overrides, we should not set `builtInImageToLoad_` to None, and instead check `iconPath_` in `getImage` first.
With this change, I think it's clearer if the `builtInImageToLoad_` is instead the moderation action type (Delete, Ban, or Timeout) which would remove the None option